cmd/agent/main.go

```diff
@@ -0,0 +1,83 @@
+// Copyright (c) 2019 Red Hat and/or its affiliates.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at:
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Main package for the Nimbess agent executable.
+package main
+
+import (
+	"flag"
+	"path/filepath"
+
+	"io"
+	"os"
+	"sync"
+
+	log "github.com/sirupsen/logrus"
+
+	"github.com/nimbess/nimbess-agent/pkg/agent"
+	"github.com/nimbess/nimbess-agent/pkg/drivers"
+	"github.com/nimbess/nimbess-agent/pkg/drivers/bess"
+)
+
+const (
+	// LogFile is name of Nimbess Agent log file
+	LogFile = "nimbess.log"
```

Comment:
Wondering if it would be better to have this nimbessAgent.log, for later on when there may be multiple nimbess x log files.

Reply:
I'll rename it to nimbess-agent.log to make it more clear.